chat/indico_chat/controllers/management.py

```diff
@@ -17,6 +17,7 @@
 from __future__ import unicode_literals
 
 from flask import session, flash, redirect, jsonify
+from flask.ext.pluginengine import render_plugin_template
```

Comment:
This is deprecated, use `flask_pluginengine` instead. Pisses me off that the PyCharm devs still didn't fix this.. https://youtrack.jetbrains.com/issue/PY-13974